dart/packages/fury-core/lib/src/code_gen/analyse/impl/annotation/fury_class_annotation_analyzer.dart

```diff
@@ -0,0 +1,54 @@
+import 'package:analyzer/dart/constant/value.dart';
+import 'package:analyzer/dart/element/element.dart';
+import 'package:fury_core/src/annotation/fury_class.dart';
+import 'package:fury_core/src/code_gen/analyse/analysis_type_identifier.dart';
+import 'package:fury_core/src/code_gen/entity/location_mark.dart';
+
+import '../../../../../fury_core.dart';
+import '../../../const/location_level.dart';
+import '../../../excep/fury_gen_excep.dart';
+import '../../annotation/location_level_ensure.dart';
+
+class FuryClassAnnotationAnalyzer {
```

Comment:
If one of the parent dir is fury, the classes can be naed without `Fury` prefix. 
And maybe `fury_class_annotation_analyzer.dart` could be renamed to `class_annotation_analyzer.dart`